packages/flutter/lib/src/cupertino/text_theme.dart

```diff
@@ -39,6 +39,20 @@ const TextStyle _kDefaultActionTextStyle = TextStyle(
   decoration: TextDecoration.none,
 );
 
+// Please update _TextThemeDefaultsBuilder accordingly after changing the default
+// color here, as their implementation depends on the default value of the color
+// field.
+//
+// Values derived from https://developer.apple.com/design/resources/.
```

Comment:
Is there a specific figma file these values were derived from? If there is, I'd recommend to link it directly.